### ISSUE TITLE
My year in commits: lessons and perspectives

### ISSUE THREAD

**Issue by ljvmiranda921** [ljvmiranda921/comments.ljvmiranda921.github.io#19]
Written on 09/04/2018 14:22:49 

URL: https://ljvmiranda921.github.io/life/2018/09/04/my-year-in-commits/

**Comment by ljvmiranda921** [comment]
Comment written by **Phuc Coi** on **12/02/2018 18:20:30**

<p>Cool nigga</p>

**Comment by sheldonhull** [comment]
@ljvmiranda921 pretty sure that comment is spam above mine, which you'd probably appreciate someone flagging. 😄

**Comment by sheldonhull** [comment]
I do like the idea of learning the stack above and below what you are doing. It's easy to be overwhelmed in the age of cloud. AWS is not easy. 
Good idea! 
I've also never thought about reviewing a commit graph to see patterns. Might do that with my blog for now. Unfortunately my current job doesn't use GithHub so I'm only in that ecosystem for personal/blog stuff right now. I do miss GitHub.⚡

**Comment by ljvmiranda921** [comment]
> @ljvmiranda921 pretty sure that comment is spam above mine, which you'd probably appreciate someone flagging. smile

Hi @sheldonhull thanks for flagging! At first I gave it some second thought but thanks for confirming that it's a spam. Will delete it now :+1:

**Comment by ljvmiranda921** [comment]
>  It's easy to be overwhelmed in the age of cloud. AWS is not easy. Good idea!

@sheldonhull Agree, even now I still find it a bit difficult. But still worth to try! 

> I've also never thought about reviewing a commit graph to see patterns. Might do that with my blog for now. 
> Unfortunately my current job doesn't use GithHub so I'm only in that ecosystem for personal/blog stuff right now.

**Would love to read yours in the future!** I was lucky back then, I was still in grad school so I was able to choose my platform. Now, we don't often use Github as much so it's hard to replicate the same blogpost every year. Similar to you, most of what I do on GIthub is personal/blog stuff!

Hope you're safe!
